docs/07-Contrib/02-sqlalchemy.md

```
@@ -0,0 +1,259 @@
+    Post,
+    aliases={"my_post_id": "post_id"},
+    strategies={"author": selectinload, "tags": selectinload},
+    max_depth=4,
+)
+
+query = SQLAlchemyQueryType([user_type, post_type])
+```
+
+
+-----
+Custom resolvers and DataLoaders
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)